File: 251009/100점 도달하기/reach-100-point.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a;
    cin >> a;
    for(int i = a; i < 101; i++) {
        if(i < 60) {
            cout << 'F' << " ";
        } else if (i < 70) {
            cout << 'D' << " ";
        } else if (i < 80) {
            cout << 'C' << " ";
        } else if (i < 90) {
            cout << 'B' << " ";
        } else if (i > 89) {
            cout << 'A' << " ";
        }
    }
    return 0;
}

File: 251028/100점 도달하기/reach-100-point.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a = int.Parse(Console.ReadLine());

        while(a <= 100) {
            if(a < 60) {
                Console.Write("F ");
            } else if(a < 70) {
                Console.Write("D ");
            } else if(a < 80) {
                Console.Write("C ");
            } else if(a < 90) {
                Console.Write("B ");
            } else {
                Console.Write("A ");
            }
            a++;
        }
    }
}
